NAMESPACE

```diff
@@ -4,6 +4,8 @@ export(annualize)
 export(getNWCData)
 export(getNWISSite)
 export(getSWECSVBlock)
+export(plotdata)
+import(ggplot2)
```

Comment:
@jnelson7 - It looks like you might have some other, uncommitted work? The changes that snuck into the NAMESPACE file are causing the package build to fail. My recommendation would be to remove the these two lines manually, commit the change and push it to github, then re-run roxygen to get the NAMESPACE back where you want it. Make sense?